.replit

```diff
@@ -1,2 +1,2 @@
 language = "cpp"
-run = "clang++ -std=c++17 -Weverything -Wno-c++98-compat -Wno-missing-prototypes main.cpp && ./a.out"
+run = "clang++ -std=c++17 -Weverything -Wno-c++98-compat -Wno-missing-prototypes main.cpp Car.cpp Highway.cpp HighwayPatrol.cpp Motorcycle.cpp SemiTruck.cpp Vehicle.cpp && ./a.out"
```

Comment:
Clear the padding warnings by adding `-Wno-padded` to the other compiler flags.
- [ ] revise